src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -12039,7 +12044,7 @@ namespace SurfaceGeometry {
 
         int countNotFullyEnclosedZones = 0;
         for (ZoneNum = 1; ZoneNum <= state.dataGlobal->NumOfZones; ++ZoneNum) {
```

Comment:
If you're not going to use `ZoneNum` for anything other than find `thisZone`, you may as well define this loop as `(thisZone : state.dataHeatBal->Zone)`. I can't believe I just suggested that. What's happening to me?